### ISSUE TITLE
TGMC Beret (Tan) missing onmob sprite.

### ISSUE THREAD

**Issue by Alexeivichi** [tgstation/TerraGov-Marine-Corps#8508]
### Description

title.

ELI5: Wearing the tan beret does not appear on head of mob for aesthetic purposes. Very important, fix please.

### Test Merges

not relevant

### Reproduction Steps

Obtain TGMC Beret (tan) as gear customization spawnables/O.S.P.
Put on character
Lose all will to live
Harm intent + aim mouth
Get bwoinked for roundstart suicide
Repost offensive and hateful copypasta to admeme
Call BowlOfCereal gay (again) in discord
Insult Xenonauts and call David Stormwell the gamer word
Never carry IronSugar for Pette
Get permenantly banned from TGMC as a whole.
Win.

**Comment by Ragecrabbo** [comment]
Skill issue
